src/main/java/io/debezium/connector/cassandra/transforms/CassandraTypeConverter.java

```diff
@@ -71,6 +71,8 @@ private CassandraTypeConverter() {
         typeMap.put(DataType.Name.TUPLE, new TupleTypeConverter());
         typeMap.put(DataType.Name.UDT, new UserTypeConverter());
         typeMap.put(DataType.Name.UUID, new BasicTypeConverter<>(UUIDType.instance));
+        typeMap.put(DataType.Name.VARINT, new BasicTypeConverter<>(IntegerType.instance));
```

Comment:
We're pretty confident in this side of the type conversion stuff - `varint` corresponds to `IntegerType` in the CQL code in C*.